tests/filecheck/dialects/arm_cf/test_ops.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: XDSL_ROUNDTRIP
+// RUN: XDSL_GENERIC_ROUNDTRIP
+// RUN: xdsl-opt -t arm-asm %s | filecheck %s --check-prefix=CHECK-ASM
+
+module {
```

Comment:
It surprises me that this verifies, as `ModuleOp` is tagged as being single_block